### ISSUE TITLE
URL d'image KO

### ISSUE THREAD

**Issue by psegalen** [chriscamicas/girr#69]
Tenter d'ajouter l'image suivante : https://cdn.hobbyconsolas.com/sites/navi.axelspringer.es/public/styles/main_element/public/filefield_paths/Nightflyers%20netflix.jpg
résulte en l'affichage suivante pour le média :
![img](https://cdn.pbrd.co/images/I07d30P.png)

**Comment by psegalen** [comment]
Le %20 peut-être ?

**Comment by mtthp** [comment]
Tout à fait. C'est parce que le nom du fichier original est réutilisé pour le sauvegarder en local mais ce n'était peut-être pas une bonne idée.

Je vais corriger ça prochainement.
